ЛогическиеОперации_Номер_2/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace ЛогическиеОперации_Номер_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+
+        {
+            var x = Input("x");
+            var y = Input("y");
+
+            Console.WriteLine("Точка принадлежит графику " + CheckGraphic(x, y));
+
+            Console.ReadKey();
+        }
+
+        static bool CheckGraphic(double x, double y)
```

Comment:
Название метода непонятное. Лучше что-то типа
```suggestion
        static bool IsInArea(double x, double y)
```